### ISSUE TITLE
pickling error while doing joblib, or numba

### ISSUE THREAD

**Issue by rsinghoyla** [intel-isl/Open3D#1448]
I get a 
TypeError: can't pickle open3d.open3d.geometry.PointCloud objects
when I try to parallelize a for loop to (multiple) query the KDTree.
I have tried joblib and numba and both give me same error.

my code is basically
```def myfunc(i):
         [k, idx, _] = pcd_tree.search_knn_vector_3d(pcd.points[i], 1000)
          return idx
results = Parallel(n_jobs=num_cores, verbose=50)(delayed(
    myfunc)(i)for i in my_list)```

Is there an alternate fast method for multiple querying the KD Tree?

**Comment by syncle** [comment]
Yes. pickling is actually [a long hanging issue](https://github.com/intel-isl/Open3D/issues/218). To querying KDTree using multi-threading, we are preparing better API for that. Please stay tuned.

**Comment by germanros1987** [comment]
Hi! In the Open3D team, we take your question very seriously. In order to provide better assistance to our users, we are implementing new policies. Now general questions should be asked in our forum: https://forum.open3d.org

Github issues will remain in use for bug reports and feature requests.

We are looking forward to answering your question in our forum!

Best regards.

**Comment by sgk-000** [comment]
Any update?
